.github/workflows/ci.yml

```
@@ -33,11 +31,11 @@ jobs:
           uv pip install --system pytest pytest-cov
           pytest --cov=src --cov-report=xml --cov-fail-under=70
       - name: Upload coverage
-        if: env.CODECOV_TOKEN != ''
+        if: ${{ secrets.CODECOV_TOKEN != '' }}
```

actionlint (1.7.7)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)